applications/app/views/printableCrossword.scala.html

```diff
@@ -0,0 +1,48 @@
+@(crosswordPage: crosswords.CrosswordPage, svg: Seq[scala.xml.Node], year: Int)(implicit request: RequestHeader)
+
+@import views.html.fragments.crosswords.printableCrosswordClue
+
+<!DOCTYPE html>
+<html>
+    <head>
+        @fragments.head(crosswordPage, None, Html(""))
+    </head>
+
+    <body class="printable-crossword">
+        <h1 class="printable-crossword__title">
+            @(crosswordPage.crossword.creator match {
+                case Some(creator) => s"${crosswordPage.crossword.name} set by ${creator.name}"
```

Comment:
ah, I was pretty much rebuilding the R2 print page, which doesn't have that - since this is part of the title of the crossword rather than metadata I think it reads better without it.